src/conn.rs

```diff
@@ -148,6 +151,10 @@ pub trait Queryable {
         where E: Into<Entid>;
 }
 
+pub trait Syncable {
```

Comment:
Put this, and the implementation of it, in `src/sync.rs`. Makes feature-flagging much easier — the `Syncable` trait would simply never exist, 'cos the `mod` import wouldn't happen.